pages/api/migration/link.ts

```diff
@@ -0,0 +1,31 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import withApiHandler from 'modules/app/api/withApiHandler';
+import { config } from 'lib/config';
+
+async function postRequestToDiscord(content: string) {
+  console.log({ content });
+  const resp = await fetch(config.MIGRATION_WEBHOOK_URL, {
```

Comment:
can we make that if the webhook is not configured returns a error message indicating that needs to add the env variable?

Reply:
Good call. Added it.